node/cli/src/command.rs

```diff
@@ -592,7 +592,7 @@ pub fn run() -> Result<()> {
 			runner.run_node_until_exit(|config| async move {
 				let extension = chain_spec::Extensions::try_get(&*config.chain_spec);
 				let para_id = extension.map(|e| e.para_id);
-				let id = ParaId::from(cli.run.parachain_id.clone().or(para_id).unwrap_or(1000));
+				let id = ParaId::from(para_id.unwrap_or(1000));
```

Comment:
We had discussed in the meeting to reintroduce the cli option `parachain_id` in moonbeam in order to be able to set the id of the test spec and local networks.

Reply:
Sure I can do that, do we finally want this @crystalin ?